pyvisa-py/highlevel.py

```diff
@@ -280,7 +312,7 @@ def write(self, session, data):
         if ret[1] < 0:
             raise errors.VisaIOError(ret[1])
 
-        return ret
+        return ret[0] # return number of bytes written
```

Comment:
This is at odd with both the docstring and the implementation of the ctypes wrapper. However the messagebased resource is actually not behaving as documented since it returns both the length and the status code contrary to what is documented. I believe that it make sense for all library call to return the library status. However consumers (Resources) should handle it and not pass it along but this should be fixed in pyvisa (MessageBased.write_raw) (but I worry it will break some people code).
So could you revert this change.

Reply:
done in d27d6f4